R/summary.R

```diff
@@ -206,25 +198,11 @@ fast_rsummry_bind <- function(lst) {
   res
 }
 
-
-
-#' summarize_rows
-#'
-#' `summarize_rows` is deprecated in favor of `make_row_df`.
-#' @param obj `VTableTree`.
-#' @return A data.frame summarizing the rows in \code{obj}.
-#' @export
-summarize_rows <- function(obj) {
-  .Deprecated("make_row_df")
-  rows <- summarize_rows_inner(obj, 0, 0)
-  fast_rsummry_bind(rows)
-}
-
-#' Summarize Rows
+#' Summarize rows
 #'
 #' @inheritParams gen_args
-#' @param depth numeric(1). Depth.
-#' @param indent numeric(1). Indent.
+#' @param depth (`numeric(1)`)\cr depth.
```

Comment:
Is it the same in {tern}? I never recall what it is exactly

Reply:
Yeah we don't have an alternative for this.